index.html

```diff
@@ -1102,6 +1102,11 @@ <h3><code>reason</code> member</h3>
       <div class="issue" data-number='117'></div>
     </section>
 
+    <section id="Internationalization">
+      <h2 id="x6-Internationalization">
+          <span its-locale-filter-list="en" lang="en"><dfn id="dfn-Internationalization">Internationalization</dfn></span>
```

Comment:
No need to add `dfn` in a section title.

Reply:
Fixed it, thank you